data/core/get_started.md

```diff
@@ -0,0 +1,175 @@
+## story_001
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch{"current_api": "dialogflow"}
+  - utter_switch_dialogflow
+
+## story_002
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch{"current_api": "luis"}
+  - utter_switch_luis
+
+## story_003
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch
+  - utter_quickstart_nlu_only
+* thank
+  - utter_noworries
+
+
+## story_004
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+
+## story_005
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch
+  - utter_quickstart_nlu_only
+* thank
+  - utter_noworries
+* out_of_scope
+  - utter_docu
+
+## story_006
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* out_of_scope
+  - utter_docu
+
+## Generated Story -5718603718660805764
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* how_to_get_started
+    - utter_quickstart
+    - utter_ask_migration
+* switch{"current_api": "luis"}
+    - utter_switch_luis
+* thank
+    - utter_noworries
+
+## Generated Story 4922905359326269181
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* how_to_get_started
+    - utter_quickstart
+    - utter_ask_migration
+* thank
+    - utter_noworries
+
+
+
+## Generated Story 6866562783165718158
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* switch{"current_api": "dialogflow"}
+    - utter_switch_dialogflow
+
+
+## tech question
+* technical_question
+  - utter_cant_answer_tech_question
+  - utter_docu
+
+## tech question again
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* switch{"current_api": "dialogflow"}
+    - utter_switch_dialogflow
+* technical_question
+  - utter_cant_answer_tech_question
+  - utter_docu
+
+
+## tech question again
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* technical_question
+  - utter_cant_answer_tech_question
+  - utter_docu
```

Comment:
the `utter_docu` utterance should be removed from the flows with `* technical_question`